x-pack/plugin/src/test/resources/rest-api-spec/api/text_structure.find_structure.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ml.find_file_structure":{
+  "text_structure.find_structure":{
```

Comment:
opted to not keep the old yaml definition (with deprecation) as the API was experimental.